justfile

```diff
@@ -35,7 +36,15 @@ output-folder := "target" / target / output-profile-folder
 output-path := output-folder / output-filename
 
 # which tool to use for compiling
-cargo-bin := if use-cargo-zigbuild != "" { "cargo-zigbuild" } else if use-cross != "" { "cross" } else { "cargo" }
+cargo-bin := if for-release != "" {
+    "cargo-auditable auditable"
```

Comment:
If we could gate it to the availability of cargo-auditable that would be cool, that way we can still have non-auditable release builds locally if we don't have the tool

Reply:
Thanks!

I've added `JUST_USE_AUDITABLE` environment variable for this.